tests/integrated/test-deploy.R

```diff
@@ -433,8 +433,19 @@ test_that("deployment timestamps respect timezone", {
   invisible(tryCatch(test_conn_1$GET_URL(myc$get_url()), error = function(e) {}))
 
   allusg <- get_usage_static(test_conn_1, content_guid = myc_guid)
+  for (i in 1:5) {
+    if (nrow(allusg) == 0) {
+      # We may need to wait a beat for the metrics to show up.
+      # Retry a few times just in case.
+      # This did not show up testing against Connect versions <= 2022.09.0,
+      # but 2023.03.0 and newer seemed to hit this
+      Sys.sleep(1)
+      allusg <- get_usage_static(test_conn_1, content_guid = myc_guid)
+    }
+  }
+  expect_equal(nrow(allusg), 1)
```

Comment:
```suggestion
  all_usage <- get_usage_static(test_conn_1, content_guid = myc_guid)
  for (i in 1:5) {
    if (nrow(all_usage) == 0) {
      # We may need to wait a beat for the metrics to show up.
      # Retry a few times just in case.
      # This did not show up testing against Connect versions <= 2022.09.0,
      # but 2023.03.0 and newer seemed to hit this
      Sys.sleep(1)
      all_usage <- get_usage_static(test_conn_1, content_guid = myc_guid)
    }
  }
  expect_equal(nrow(all_usage), 1)
```

I know you weren't changing this, but when trying to see what this was about it took me a click to figure out what `allusg` was

Reply:
Took care of this separately, had to change the next line too